pkg/windows/windows.go

```diff
@@ -140,6 +145,11 @@ func parseNetUseOutput(text string) (remoteDrives, error) {
 			continue
 		}
 
+		status := strings.TrimSpace(line[:cols.Local.Start])
+		if status != "" && strings.ToLower(status) != "ok" {
+			continue
+		}
+
 		local := line[cols.Local.Start : cols.Local.Start+cols.Remote.Width]
 		local = strings.ToUpper(strings.TrimSpace(local))
 		letter := strings.Split(local, ":")[0][0]
```

Comment:
If the drive letter is empty, this will result in a panic:
`panic: runtime error: index out of range [0] with length 0`

We need to check the length of `strings.Split(local, ":")` and `strings.Split(local, ":")[0]` before trying to take the first char.

Reply:
It would never happen because a drive letter is always linked to a remote path and vice-versa?

Reply:
In searching online i'm seeing examples where the drive letter is empty. Doesn't hurt to guard against it.

Reply:
it's already safe. Take a look at the [new test added](https://github.com/wakatime/wakatime-cli/pull/618/commits/1f0148e9fd63d681137e37aee9a3c54cbe892048#diff-d344b69e095a2125ed47fb159fbd42d2ef90c351badecd786c1514d6399b51ffR27).